includes/migration-guide/retargeting/versionselector.md

```diff
@@ -1,4 +1,5 @@
 > [!div class="op_multi_selector" title1="From .NET Framework Version" title2="To .NET Framework Version"]
```

Comment:
Also confirm these work on the review site

Reply:
They do.

Reply:
Thanks!